app/views/entities/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render 'devise/shared/navbar' %>
+<div class="group-wrapper">
+  <% if @entities.empty? %>
+    <p class="no-trans">No transactions yet</p>
+  <% else %>
+    <% @entities.each do |entity|%>
+      <%= link_to(entity_path(entity)) do %>
+        <ul class="group-container">
+          <li>
+              <img class='img' src="<%= entity.group.icon %>" alt="icon">
+              <h3><%= entity.name %></h3>
+              <%= entity.created_at.to_date.to_s %>
+            <h4 class="group-amount">
+              <%= entity.amount %>
+            </h4>
+          </li>
+        </ul>
```

Comment:
- [ ] For each category, the user should  see their name, icon and the total amount of all the transactions that belongs to that category, kindly add user name